src/helpers/AuthHelper.ts

```diff
@@ -1 +1,36 @@
-export {};
+import config from '../config/environment'
+import { User } from '../interfaces/UserInterface'
+import jwt from 'jsonwebtoken'
+
+export class TokenGen {
+  payload: User
+  constructor(payload: User) {
+    this.payload = payload
+  }
+
+  create() {
```

Comment:
create what exactly! come up with a better naming

also it should take the payload as a parameter to sign it and generate the token.